draft-ietf-ipsecme-ikev2-sa-ts-payloads-opt.md

```diff
@@ -145,9 +145,9 @@ HDR, SK {N(OPTIMIZED_REKEY,newSPIi),
 
 # Optimized Rekey of Child SAs
 
-The initiator of an optimized rekey request sends a CREATE_CHILD_SA payload with the OPTIMIZED_REKEY notify payload containing the new Security Parameter Index (SPI) for the new Child SA. It omits the SA and TS payloads. If the current Child SA was negotiated with Perfect Forward Secrecy (PFS), a KEi payload MUST be included as well. If no PFS was negotiated for the current Child SA, a KEi payload MUST NOT be included.
+The initiator of an optimized rekey request sends a CREATE_CHILD_SA request with the OPTIMIZED_REKEY notify payload containing the new Security Parameter Index (SPI) for the new Child SA. It omits the SA and TS payloads. If the current Child SA was negotiated with Perfect Forward Secrecy (PFS), a KEi payload MUST be included as well. If no PFS was negotiated for the current Child SA, a KEi payload MUST NOT be included.
 
-The responder of an optimized rekey request performs the same process. It includes the OPTIMIZED_REKEY notify with its new IKE SPI and omits the SA and TS payloads. Depending on the PFS negotiation of the current Child SA, the responder includes a KEr payload.
+The responder of an optimized rekey request performs the same process. It includes the OPTIMIZED_REKEY notify with its new SPI for the new Child SA and omits the SA and TS payloads. Depending on the PFS negotiation of the current Child SA, the responder MUST include a KEr payload.

```

Comment:
Why "MUST include a KEr payload"? The corresponding text for initiator describes two cases - MUST include with PFS, and MUST NOT include with no PFS. The text for responder seems to only conside PFS case. I think that either the initiator's text should be copy-pasted here or "MUST" is removed and a neutral original text ("Depending on PFS ... includes ...") persists.

Reply:
The original comment was to keep the normative language being used in the same way for both the initiator and responder. If there is no objection, I can delete the "MUST" and "MUST NOT" in the initiator's part.

Reply:
If people prefer keeping the normative language, I'm also fine with that and I'll do the corresponding update.

Reply:
I'm fine either way, but please do this for both initiator and responder, so that the text is consistent.